kanidm_proto/src/v1.rs

```diff
@@ -98,6 +98,92 @@ pub enum PasswordFeedback {
     BadListed,
 }
 
+/// Human-readable PasswordFeedback result.
+impl fmt::Display for PasswordFeedback {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        match self {
+            PasswordFeedback::AddAnotherWordOrTwo => write!(f, "Add another word or two."),
+            PasswordFeedback::AllUppercaseIsAlmostAsEasyToGuessAsAllLowercase => write!(
+                f,
+                "All uppercase is almost as easy to guess as all lowercase."
+            ),
+            PasswordFeedback::AvoidDatesAndYearsThatAreAssociatedWithYou => write!(
+                f,
+                "Avoid dates and years that are associated with you or your account."
+            ),
+            PasswordFeedback::AvoidRecentYears => write!(f, "Avoid recent years."),
+            PasswordFeedback::AvoidRepeatedWordsAndCharacters => {
+                write!(f, "Avoid repeated aords and characters.")
+            }
+            PasswordFeedback::AvoidSequences => write!(f, "Avoid sequences of characters."),
+            PasswordFeedback::AvoidYearsThatAreAssociatedWithYou => {
+                write!(f, "Avoid years that are associated with you.")
+            }
+            PasswordFeedback::AWordByItselfIsEasyToGuess => {
+                write!(f, "A word by itself is easy to guess.")
+            }
+            PasswordFeedback::BadListed => write!(f, "This password has been 'Bad Listed'."),
```

Comment:
Perhaps "This password has been compromised and can not be used" ?

Reply:
Maybe "This password is blocked or may have been compromised and can not be used." because there's multiple reasons to badlist?

Reply:
Yeah, that works for me :)